### ISSUE TITLE
Show most recent Form 1 for committees

### ISSUE THREAD

**Issue by lbeaufort** [fecgov/fec-cms#2964]
Show most recent Form 1 for committees on the "about this committee tab" like we show most recent F2 for candidates on the "About this candidate" tab.

Example of F2 most recent implementation:

![Screen Shot 2019-09-19 at 11 43 51 AM](https://user-images.githubusercontent.com/12799132/65259434-a81d7180-dad2-11e9-9a66-491925ee32c6.png)

Example of "About this committee" page
![Screen Shot 2019-10-17 at 9.57.00 AM.png](https://images.zenhubusercontent.com/59a579f28f62dc7798c47359/a5077d95-c6f1-4f11-a314-784d8fe69b7e)

**Completion criteria:**
- [x] Timebox a design that resembles what happens on About this candidate tab
- [x] Implement changes according to mock-up

**Comment by PaulClark2** [comment]
Closing in favor of https://github.com/fecgov/fec-cms/issues/3079

**Comment by PaulClark2** [comment]
Wrong ticket. Reopening.

**Comment by lbeaufort** [comment]
@JonellaCulmer how is this placement for most recent Statement of Organization? Reference image for candidates in the original issue text above. 

![Screen Shot 2019-10-25 at 10.59.26 AM.png](https://images.zenhubusercontent.com/59a579f28f62dc7798c47359/eb947db1-e856-4e95-bea2-3a40b96e254f)

**Comment by JonellaCulmer** [comment]
Looks great! Thanks @lbeaufort

**Comment by lbeaufort** [comment]
This issue is blocked by https://www.github.com/fecgov/openfec/issues/4016 because the changes to the committee profile pages for that issue are extensive enough that it makes sense to rebase this branch after that work is merged.

**Comment by JonellaCulmer** [comment]
@lbeaufort https://github.com/fecgov/openFEC/pull/4039 has been merged. Is this now safe to merge as well?
